DLaB.AttributeManager/AttributeManagerPlugin.cs

```diff
@@ -1043,6 +1146,35 @@ private void SelectAllKeyDownHandler(object sender, KeyEventArgs e)
                 e.Handled = true;
             }
         }
+
+        private void grpSteps_Enter(object sender, EventArgs e)
```

Comment:
Can you remove this?

Reply:
This looks like an accidental creation while editing the UI. I have removed in latest changes.